### PR TITLE
Add installation with Homebrew section in README

### PR DESCRIPTION
**Pull request by abitrolly**
Added `web-ext` to Homebrew here https://github.com/Homebrew/homebrew-core/pull/142942

### CODE REVIEW

**Comment by VersoriumX** [comment]
reviewed

**Review by VersoriumX — APPROVED** [review]

**Comment by abitrolly** [comment]
@willdurand 8.0.0 is uploaded and updated automatically.

Are there any objections against having this as an official installation method?

**Comment by willdurand** [comment]
> Are there any objections against having this as an official installation method?

Yes, but if you mark it as "unofficial", I suppose that can end up in the readme.

File: README.md [hunk 1]
```diff
@@ -31,7 +31,15 @@ Here are the commands you can run. Click on each one for detailed documentation
 - [`docs`](https://extensionworkshop.com/documentation/develop/web-ext-command-reference#web-ext-docs)
   - Open the `web-ext` documentation in a browser
 
-## Installation from npm
+## Installation
+
+### Using Homebrew
```
Comment (willdurand) [inline README.md:36]: Please move that section right before "Installation from source" and add:

```suggestion
### Using Homebrew (unofficial)

The community maintains a `web-ext` formula.
```

Comment (abitrolly) [reply README.md]: Done.

**Review by willdurand — APPROVED** [review]

**Pull request merged**